examples/ui/ui_texture_atlas_slice.rs

```diff
@@ -50,7 +50,8 @@ fn setup(
     mut texture_atlases: ResMut<Assets<TextureAtlasLayout>>,
 ) {
     let texture_handle = asset_server.load("textures/fantasy_ui_borders/border_sheet.png");
-    let atlas_layout = TextureAtlasLayout::from_grid(UVec2::new(50, 50), 6, 6, None, None);
+    let atlas_layout =
```

Comment:
Looking at the image, this is probably correct, but is unrelated to the other changes.

Reply:
It is but the slicer `border_rect` should be increased from `22` to `24` @s-puig , the fifth texture in the sheet is drooling on the border